src/cc/libbpf.c

```diff
@@ -589,7 +589,7 @@ int bpf_open_perf_event(uint32_t type, uint64_t config, int pid, int cpu) {
   return fd;
 }
 
-int bpf_attach_xdp(const char *dev_name, int progfd) {
+int bpf_attach_xdp(const char *dev_name, int progfd, __u32 flags) {
```

Comment:
nit, but can you please use uint32_t here to follow the style of the remainder of the file?